src/hypothesis/internal/conjecture/engine.py

```diff
@@ -1527,6 +1528,54 @@ def reoffset(o):
 
         minimize_int(offset, reoffset)
 
+    def shrink_offset_pairs(self):
+        """Lower any two blocks offset from each other the same ammount.
+
+        Try lowering every non-zero pair of non-shrinking blocks by a
```

Comment:
I think this docstring is saying the same thing twice?

It would be nice to have an example in here of the sort of shrinks this enables (just something based on your PR comment would be fine)